### PR TITLE
Remove big from core utils

### PR DESCRIPTION
**Pull request by dimriou**
`Big` implementation is EVM-specific, but it's been used across our codebase via the core utils package. In order to move the EVM-specific code to a separate relay we have to move `Big` under the evm package and import it wherever is needed. If users don't want to do that, they can always utilize the standard `math/big`.

### CODE REVIEW

File: core/chains/evm/utils/big/big.go [hunk 1]
```diff
@@ -51,7 +52,7 @@ func (b *BigFloat) Value() *big.Float {
 type Big big.Int
 
 // NewBig constructs a Big from *big.Int.
```
Comment (dimriou) [inline core/chains/evm/utils/big/big.go:54]: Fix name

File: core/chains/evm/utils/big/big.go [hunk 2]
```diff
@@ -61,8 +62,8 @@ func NewBig(i *big.Int) *Big {
 }
 
 // NewBigI constructs a Big from int64.
```
Comment (dimriou) [inline core/chains/evm/utils/big/big.go:64]: Fix name

File: core/services/relay/evm/types/types_test.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
 )
 
-// ChainID   *utils.Big `json:"chainID"`
+// ChainID   *big.Big `json:"chainID"`
```
Comment (dimriou) [inline core/services/relay/evm/types/types_test.go:15]: Fix indentation

**Review by samsondav — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Review by krehermann — COMMENTED** [review]
Can you add a description to the PR comment about why we need this and/or a jira ticket reference?

i'm sure it's related to moving out the evm code to it's own repo, but it's not immediately clear to me why this change moves in the right direction because there are a lot of files out of the `evm` tree that now depend on `chains/evm` via the ubig.

**Comment by dimriou** [comment]
> Can you add a description to the PR comment about why we need this and/or a jira ticket reference?
> 
> i'm sure it's related to moving out the evm code to it's own repo, but it's not immediately clear to me why this change moves in the right direction because there are a lot of files out of the `evm` tree that now depend on `chains/evm` via the ubig.

Added a description above

**Review by krehermann — APPROVED** [review]

**Pull request merged**